src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1142,7 +1142,12 @@ private void InitializeNodeLibrary(IPreferences preferences)
             foreach (var path in pathManager.DefinitionDirectories)
             {
                 // NOTE: extension will only be null if path is null
-                string extension = Path.GetExtension(path);
+                string extension = null;
+                try
+                {
+                    extension = Path.GetExtension(path);
+                }
+                catch (ArgumentException) { }
```

Comment:
should we log the error here to the dynamo console?

Reply:
Logging of the ArgumentException message added